Src/FluentAssertions/Primitives/DateTimeOffsetAssertions.cs

```diff
@@ -68,7 +68,7 @@ public DateTimeOffsetAssertions(DateTimeOffset? value)
         }
 
         /// <summary>
-        /// Asserts that the current <see cref="DateTimeOffset"/> is exactly equal to the <paramref name="expected"/> value.
+        /// Asserts that the current <see cref="DateTimeOffset"/> represents the same point in time as the <paramref name="expected"/> value.
```

Comment:
🤔 Shouldn't we also update the failure message to cover this important distinction?

Reply:
Done. However, the messages became a bit wordy. Perhaps too much so.

Reply:
Maybe, but it'll make it very clear why the test failed.